minidec/main.rs

```diff
@@ -59,6 +67,7 @@ fn main() {
         //println!("Local Variable info: {:?}", rfn.locals);
         let mut writer: IRWriter = Default::default();
         writer.emit_il(Some(rfn.name.clone()), &rfn.ssa, &mut ff);
+        writer.emit_il(Some(rfn.name.clone()), &rfn.ssa, &mut ffm);
```

Comment:
Instead of doing it this way, please write the output from the first call to `emit_il` into a string like:

```rust
...
write.emit_il(Some(rfn.name.clone()), &rfn.ssa, &mut res));
...
```

And then use that string to write out to two files. One, the function specific file (`ff`) and the other, the new `ffm`. I could do this, but I'll let you give this a try first.